src/test/java/net/openhft/chronicle/core/internal/threads/VanillaThreadLockTest.java

```diff
@@ -93,7 +93,7 @@ public void lockTimeOut() {
         VanillaThreadLock lock = new VanillaThreadLock(value, 50);
         lock.lock(-1);
 
-        expectException("ThreadId -1 died while holding a lock");
+        ignoreException("ThreadId -1 died while holding a lock");
```

Comment:
why did this change?

Reply:
So prior to this change `expectException` effectively meant `ignoreException`. I didn't change most of them (i.e. they have been upgraded to expectations), but some, the ones I changed in this PR didn't occur on either my machine or one of the other platform builds.

Reply:
I thought expectException meant "this exception will happen, ignore it, and throw if it does not happen" and ignoreException means "this exception may happen, ignore it"

Reply:
Yeah that's what it means once you set it up that way (which I did in this PR), but in a lot of projects (including this one previously) expectException meant "this exception may happen, ignore it"

That's why I had to change some expects to ignores, because they either don't happen anywhere or don't happen on one of the platforms we test on

Reply:
ah got it - nice